frontend/src/pages/auth/components/auth-layout/auth-layout.tsx

```diff
@@ -0,0 +1,40 @@
+import { type ReactNode } from 'react';
+
+import {
+  AUTH_FORM_WRAPPER_FLOATING_LETTERS,
+  HERO_FLOATING_LETTERS,
+} from './libs/constants.js';
+import { type FloatingLetter } from './libs/types/floating-letter.type.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  formElement: ReactNode;
+};
+
+const AuthLayout: React.FC<Properties> = ({ formElement }) => {
+  const renderFloatingLetters = (letters: FloatingLetter[]): ReactNode[] =>
+    letters.map((item) => (
+      <div
+        key={item.value}
```

Comment:
Using item value (which is letter), may lead to some unexpected behavior - as these letters may be the same (like duplicating W and R) and duplicate keys for elements may cause something bad with rendering